demo/main.js

```diff
@@ -1561,6 +1561,20 @@ shakaDemo.Main = class {
         request.contentSourceId = asset.imaContentSrcId;
         request.videoId = asset.imaVideoId;
       }
+      switch (asset.imaManifestType) {
+        case 'DASH':
+        case 'dash':
+        case 'MPD':
+        case 'mpd':
+          request.format = 'dash';
```

Comment:
Man, I had to actually look through the code to verify that the string values of these enums are in lowercase. The public documentation only says that the enums are `StreamRequest.StreamFormat.DASH` and `StreamRequest.StreamFormat.HLS`.

Reply:
I am going to change it, but the cases of the case are for the value that the user enters in the popup, and here it is transformed to the value that DAI expects.

Reply:
Yeah, I know.
I was just commenting, since I was trying to cross-reference with the IMA SDK docs to make sure the values you put in were right. I was disappointed that the docs were so vague about that enum.